### PR TITLE
Parallelize `loadTransaction` in the GraphQL resolver

### PR DESCRIPTION
**Pull request by steveluscher**
# Summary

This code makes a backup fetch for `jsonParsed` every time you fetch a transaction. Previous to this PR those fetches were done in serial with any existing `base58`/`base64`/`json` fetches.

### CODE REVIEW

**Comment by steveluscher** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #2059** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2059?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2059?utm_source=stack-comment).

**Review by buffalojoec — APPROVED** [review]
Nice!

File: packages/rpc-graphql/src/loaders/transaction.ts [hunk 1]
```diff
@@ -30,39 +30,26 @@ function normalizeArgs({ commitment = 'confirmed', encoding = 'jsonParsed', sign
...
-    const transaction = await rpc
-        .getTransaction(
-            signature,
-            // @ts-expect-error FIXME: https://github.com/solana-labs/solana-web3.js/issues/1984
-            config,
-        )
-        .send()
-        .catch(e => {
-            throw e;
-        });
```
Comment (steveluscher) [inline packages/rpc-graphql/src/loaders/transaction.ts:42]: `await` throws errors naturally from this callsite. Did you find something that made it necessary to explicitly supply a `catch` callback @buffalojoec?

Comment (buffalojoec) [reply packages/rpc-graphql/src/loaders/transaction.ts:42]: I don't actually remember specifically why I wrote a `catch` there, but I know some RPC errors weren't showing up in the tests when I was debugging, and I'm almost certain I also has `--verbose` passed to Jest.

**Pull request merged**